docs/CONTRIBUTING.md

```diff
@@ -98,43 +120,37 @@ Also, if you add an empty file, named `__init__.py` at tests folder, you'll be a
 
 (e.g.: `python -m unittest tests.test_bounding_box_query`)
 
-
-
 **Pull Requests**
 
-Code review comments may be added to your pull requests, keep an eye out for necessary changes before merge approval. If the maintainer decides that your code is good, it will be merged to ANYWAY's Dev repository.
-On your pull requests, don't forget to reference to the fixed bug / feature by adding `Closes #XXXX` or `Fixes #XXXX` to the body of the requests. Also, shortly summarize the changes made and list any
```

Comment:
either "reference the" or "add a reference to"

Reply:
it's in deleted section

Reply:
@elazarg thanks, I solved the reference sentences grammar